src/data/nat/factorial.lean

```diff
@@ -170,12 +170,12 @@ lemma succ_desc_fac (n : ℕ) : ∀ k, (n + 1) * desc_fac n.succ k = (n + k + 1)
 | 0 := by rw [add_zero, desc_fac_zero, desc_fac_zero]
 | (k + 1) := by rw [desc_fac, mul_left_comm, succ_desc_fac, desc_fac, succ_add, ← add_assoc]
 
-/-- Prove that `desc_fac` is what it is promised to be. Stated divison-less for ease. -/
+/-- `desc_fac n k = (n + k)! / n!`. Stated divison-less for simpicity. -/
```

Comment:
```suggestion
/-- `desc_fac n k = (n + k)! / n!`. However, this lemma states a reformulation to avoid `nat`-division.
See `eval_desc_fac'` if you really need the version that uses `nat`-division. -/
```